### DISCUSSION TITLE
Π2019166 - Παραδοτέο 6

### DISCUSSION THREAD

**Discussion by olumpiaalexopoulou** [courses-ionio/sw#1479] [Show and tell]
Σε αυτό το παραδοτέο μέσω του Alpine linux αποφάσισα να χρησιμοποιήσω το bashblog για να μπορέσω να δημιουργήσω ένα post για το static site blog μου. 

Links:
 - [Asciinema Link](https://asciinema.org/a/q191tpLd3et0cJrVhks4ggAX5)
 - [My repository](https://github.com/olumpiaalexopoulou/oliv.github.io)

[Η Αναφορά μου](https://github.com/olumpiaalexopoulou/sw/tree/2019166/projects/2019166)

**Comment by p20sami** [comment]
@olumpiaalexopoulou Σχετικά με το πρόβλημα που αναφέρεις στην αναφορά, κατα πάσα πιθανότητα φταίει το όνομα του repository γιατί το πρώτο μέρος πρεπει να είναι ακριβώς όπως το username (Βήμα 1: https://pages.github.com/). Ελπίζω να βοηθήσει!

**Reply by olumpiaalexopoulou** [reply]
Σε ευχαριστώ πολύ θα το δοκιμάσω και έτσι!
